utils/update-checkout-config.json

```diff
@@ -9,7 +9,7 @@
         "swift": {
             "remote": { "id": "apple/swift" } },
         "lldb": {
-            "remote": { "id": "apple/swift-lldb" } },
+            "remote": { "id": "marcrasi/swift-lldb" } },
```

Comment:
I pointed these at my forks of the repos, because I don't have commit access to the original repos yet. I'll point these back to the original repos before I merge this PR.